APMrover2/Rover.h

```diff
@@ -490,6 +491,15 @@ class Rover : public AP_HAL::HAL::Callbacks {
 
     // Simple mode
     float simple_sin_yaw;
+
+    enum FS_Options {
```

Comment:
Let's move this up below the "enum Failsafe_Action" and rename it to "Failsafe_Options" to be consistent.

Reply:
done